### DISCUSSION TITLE
Optimising Stellarium for 4K projection?

### DISCUSSION THREAD

**Discussion by ImmersiveTheatres** [Stellarium/stellarium#2500] [Q&A]
Hi everyone - I've just purchased a 4K projector for my mobile planetarium, and Stellarium looks AWESOME on the dome! Yet this is without changing anything in the config file. Are there things I can do to optimise Stellarium even more for 4K (3840 x 2160)? eg. how do I get higher resolution planet textures (they still don't look very good).

Currently the video section of my config file looks like this (as it has always been for my old HD projector):

[video]
fullscreen                             = true
maximum_fps                      = 10000
minimum_fps                       = 18
screen_h                              = 702
screen_number                   = 1
screen_w                             = 1071
screen_x                              = 237
screen_y                              = 85
viewport_effect                   = sphericMirrorDistorter

I've had a look, but can't find any documentation describing how to make Stellarium look better at higher screen resolutions.

Thanks very much
Mario

**Comment by gzotti** [comment]
We do not provide higher resolution textures, although it is easy to replace those that we have with images you like better.

**Reply by ImmersiveTheatres** [reply]
Great, thanks for the tip. 

Now, I've been searching for half an hour online, and I can't find appropriate high-res textures for Stellarium planets / Milky Way, etc. Just a bunch of old discussions and dead links. 

Does anyone know where I can find a fresh repository of high-res textures please?

**Reply by gzotti** [reply]
Google for "NASA planet textures" for a start...

**Comment by jasubhai** [comment]
Hello,

Please could you guide me on the following; I would like to use Stellarium to project the night sky on my bedroom ceiling. Currently, I do not have any equipment nor do I know how to do so. 

I'm assuming I would be able to use a standard projector for the above, and if so, would prefer the same as then I could also use it as a video projector.

If you know a better way than with or without using Stellarium - like home planetariums eg Sega Homestar flux etc. Please let me know the same.

Thank You,
Jasu Bhai

**Reply by ImmersiveTheatres** [reply]
The way I project Stellarium in my mobile planetariums is by using a special spherical mirror (http://paulbourke.net/dome/mirrordome/). 

The mirror is of course not completely spherical, but only a segment of a sphere (http://bit.ly/ase-sm). Note: it may look like a regular security mirror, but it is in fact 'first-surface' coated ie. the reflective silver does not have a transparent protective layer over it (as this will cause refraction of light and double images). These mirrors are therefore quite expensive at approx. Euro 800.

The projector needs to be a short throw projector (to be able to focus on the mirror). You can see recommended projectors listed here: http://paulbourke.net/dome/faq/#projectors. I highly recommend spending a little more and buying a 4K projector. Stellarium looks brilliant in 4K!

**Comment by jasubhai** [comment]
I guess this arrangement would be necessary if I'm projecting onto a dome.However I will be projecting on a flat surface - my bedroom ceiling. In this case, I assume a dome mirror & a short focus projector are not necessary - is that correct?  Would you be knowing of a good way to project it on the ceiling, my objective is to achieve the feeling of sleeping under Stars. And prefer the projection to cover a human eye  FOV and be in real time for a given location. Thank you--Sent from my phone using dictation. Please excuse my brevity, typos & other writing errors. Thank You.On 1/11/23, 12:09 AM Mario Di Maggio ***@***.***> wrote:

   
  The way I project Stellarium in my mobile planetariums is by using a special spherical mirror (http://paulbourke.net/dome/mirrordome/). 
  The mirror is of course not completely spherical, but only a segment of a sphere (http://bit.ly/ase-sm). Note: it may look like a regular security mirror, but it is in fact 'first-surface' coated ie. the reflective silver does not have a transparent protective layer over it (as this will cause refraction of light and double images). These mirrors are therefore quite expensive at approx. Euro 800. 
  The projector needs to be a short throw projector (to be able to focus on the mirror). You can see recommended projectors listed here: http://paulbourke.net/dome/faq/#projectors. I highly recommend spending a little more and buying a 4K projector. Stellarium looks brilliant in 4K!

**Reply by gzotti** [reply]
I guess for this a super-quiet projection is mandatory. Most projectors have a loud fan. If you don't care for the actual functions of Stellarium, such a simple star ball even at children toy level may be sufficient if quiet. It's even cheaper than an average video projector. However I find nothing about its noise level. And I have never seen it in practice, so cannot further comment.

**Reply by ImmersiveTheatres** [reply]
It doesn't need to be a dome. I've projected with the spherical mirror in standard rooms with standard ceilings before, and it works very well.

It's true though, projectors can be quite loud (especially if you're trying to fall asleep). 

Yet I would say be careful of the silly 'stars on your ceiling' toys. At the last planetarium I worked at we tested at least half a dozen of them (the museum gift shop was thinking of stocking them) - and they were all RUBBISH. Absolute junk, that's what many of these toys are; producing blobs of light on the ceiling that doesn't look anything like the night sky. So maybe test before you buy? (or at least have the freedom to return it if you don't like it).

**Comment by jasubhai** [comment]
Thank you for your suggestion.  Actually I am considering sega's HomeStar flux, however this and similar devices devices project slides in other words they are static images. I'm hoping for something that will project the sky in real time and within the human eye fov.So it seems using one of the above devices would be the appropriate option for my needThanks ****Sent from my phone using dictation. Please excuse my brevity, typos & other writing errors. Thank You. On 1/11/23, 11:07 AM Georg Zotti ***@***.***> wrote:

   
  I guess for this a super-quiet projection is mandatory. Most projectors have a loud fan. If you don't care for the actual functions of Stellarium, such a simple star ball even at children toy level may be sufficient if quiet. It's even cheaper than an average video projector.
